docs/pages/debugging/devtools-plugins.mdx

```diff
@@ -176,6 +176,36 @@ export default function App() {
 
 In the terminal, run `npx expo start`, press <kbd>shift</kbd> + <kbd>m</kbd> to open the list of dev tools, and then select the React Query plugin. This will open the plugin's web interface, displaying queries as they are used in your app.
 
+### Redux DevTools
+
+Based on the full [Redux DevTools](https://github.com/reduxjs/redux-devtools/) (from the Chrome extension) but as an Expo Dev Plugin.
+
+- Easy to install and run.
+- Live list of actions and how it affects the state.
+- Ability to rewind, replay, and dispatch actions from the devtools.
+
+To use the plugin, start by installing the package:
+
+<Terminal cmd={['$ npx expo install redux-devtools-expo-dev-plugin']} />
+
+If you're using `@reduxjs/toolkit` then you need to modify your call to `configureStore` to disable the built in devtools (by passing in `devTools: false`) and add in the Expo Devtools plugin enhancer (by concatenating the `devToolsEnhancer()`).
+
+Your call to `configureStore` will end up looking like this:
```

Comment:
```suggestion
If you're using `@reduxjs/toolkit`, modify the `configureStore` call to disable the built-in dev tools by passing in `devTools: false`. Then, add in the Expo DevTools plugin enhancer by concatenating the `devToolsEnhancer()`. The `configureStore` call is going to look like the following:
```

Let's also update the above paragraph. Instead of using brackets, since those user-facing changes, let's make part of the main sentences and also split the sentences here for better readability.